docs/docsite/rst/getting_started_ee/introduction.rst

```diff
@@ -0,0 +1,65 @@
+.. _introduction_execution_environments:
+
+######################################
+Introduction to execution environments
+######################################
+
+Ansible execution environments aim to resolve complexity issues and provide all the benefits you can get from containerization.
+
+Reducing complexity
+===================
+
+There are three main areas where EEs can reduce complexity: software dependencies, portability, and content separation.
+
+Dependencies
+------------
+
+Software applications typically have dependencies.
+It can be software libraries, configuration files or other services, to name a few, and Ansible, being an exceptional automation tool, is not an exception in terms of the mentioned.
+
+Traditionally, application dependencies are installed by administrators on top of an operating system using packaging management tools like RPM or Python-pip.
```

Comment:
```suggestion
Traditionally, administrators install application dependencies on top of an operating system using packaging management tools such as RPM or Python-pip.
```

Reply:
We avoid using 'like' because it doesn't translate easily. The other change here makes the sentence 'active' instead of 'passive'.  `are installed by` or similar phrases are a good way to notice passive voice.  The fix is to change a sentence so it starts with who is doing something. "The EE guide was written by @Andersson007 " is passive.  "@Andersson007 wrote the EE guide" is active. it shows you doing something :-)

Reply:
- `like`: cool, didn't know, thanks
- `active-passive`: yep, i know how it works:D i a bit forget about it, now i remember that it's mentioned in the style guide, thanks for reminding!